### DISCUSSION TITLE
How to improve Cmder startup time?

### DISCUSSION THREAD

**Discussion by quazi-irfan** [cmderdev/cmder#2587] [Q&A] [answered]
It takes about 7-8 seconds for Cmder to startup on my Windows 10 machine.

It's a fresh Windows and Cmder installation. Is there any Cmder settings I can tweak to speed up the startup? I'd like to have it in <2 seconds.

Thanks!

**Comment by mrx23dot** [comment, marked as answer]
In short empty init.bat 's content,
or keep what you need.
Instant startup.
https://github.com/cmderdev/cmder/issues/2602#issuecomment-918229335

**Reply by quazi-irfan** [reply]
If anyone is wondering, this file can be found here: `<Cmder installation>/vendor/init.bat`
